src/main/resources/i18n/Parsing_de.properties

```diff
@@ -10,20 +10,17 @@
 # REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
 # so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
 #
-# Prior to Java 9, properties files are encoded in ISO-8859-1.
-# We have to use \u00fc instead of the German ue character, \u00e4 for ae, \u00f6 for oe, \u00df for ss
-#
-InvalidSyntax.noMessage=Ung\u00fcltige Syntax in Zeile {0} Spalte {1}
-InvalidSyntax.full=Ung\u00fcltige Syntax, ANTLR-Fehler ''{0}'' in Zeile {1} Spalte {2}
+InvalidSyntax.noMessage=Ung�ltige Syntax in Zeile {0} Spalte {1}
```

Comment:
GitHub diff preview doesn't support UTF-8. Shakes fist at ASCII...

I promise you the umlauts are there. On the top right of this file in the diff, click on the ... and then click "View File" to see the full file, umlauts included.

e.g. https://github.com/graphql-java/graphql-java/blob/0428ebb85c38a655a4e4b1d3bfa897bbd48b94b0/src/main/resources/i18n/Parsing_de.properties